cli/utils_for_test.go

```diff
@@ -143,6 +143,7 @@ func withMockStdout(t *testing.T, operation func(*os.File) error) error {
 // waitForRESTService waits until the REST service becomes available to serve
 // requests or the context times out.
 func waitForRESTService(ctx context.Context, t *testing.T, url string) {
+	client := &http.Client{}
```

Comment:
can you put a sync.Pool in for this so that we can reuse clients?

Reply:
Done. All tests that need http clients take them from the pool.